client/src/pages/Requests/requests.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import axios from 'axios';
+
+class Request extends React.Component {
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  constructor(props: {} | Readonly<{}>) {
+    super(props);
+    this.state = {
+      requestInfo: null,
+      error: '',
+    };
+  }
+
+  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+  getRequestInfo = () => {
+    axios
+      .get('/request')
+      // eslint-disable-next-line @typescript-eslint/ban-types
+      .then((response: { data: object }) => {
+        const data = response.data;
+        console.log(data);
+      })
+      .catch((error: unknown) => {
+        console.log(error);
+      });
```

Comment:
Could we utilise async/await, for project consistency. Also remove the unecessary console logs. If something fails grabbing the requests it's useless for a user to have a console log, we should be sending snackbar notifications with the `useSnackbar` hook included with the project.